titan/model.py

```diff
@@ -235,7 +235,13 @@ def update_all_agents(self):
         # If static network, ignore relationship progression
         if not self.params.features.static_network:
             for rel in copy(self.pop.relationships):
-                if rel.progress():
+                if (
+                    self.params.partnership.dissolve.enabled
+                    and self.time == self.params.partnership.dissolve.time
+                ):
+                    rel.progress(force=True)
+                    self.pop.remove_relationship(rel)
+                elif rel.progress():
                     self.pop.remove_relationship(rel)
```

Comment:
Can we dedent `self.pop.remove_relationship(rel)` since it happens in both conditions?

```suggestion
                if (
                    self.params.partnership.dissolve.enabled
                    and self.time == self.params.partnership.dissolve.time
                ):
                    rel.progress(force=True)
                self.pop.remove_relationship(rel)
```

Reply:
Oh wait, it's not an else. Forget this comment it's an elif